jaeger_client/config.py

```diff
@@ -84,17 +84,20 @@ class Config(object):
     _initialized_lock = threading.Lock()
 
     def __init__(self, config, metrics=None, service_name=None, metrics_factory=None,
-                 validate=False):
+                 validate=False, scope_manager=None):
         """
         :param metrics: an instance of Metrics class, or None. This parameter
             has been deprecated, please use metrics_factory instead.
         :param service_name: default service name.
             Can be overwritten by config['service_name'].
         :param metrics_factory: an instance of MetricsFactory class, or None.
+        :param scope_manager: a class implementing a scope manager, or None for
```

Comment:
why should this arg be a class rather than an actual object?

Reply:
for no good reason it seems, looking at it now I'd say that passing an object would probably better since it's more flexible. I'll try it out and see if it doesn't break anything.

Reply:
thanks. Passing object makes more sense to me since it's supposed to be a singleton, so there's little reason why the tracer should be instantiating it from a class.

Reply:
Makes sense, the tests pass with this change.